external/emulation/emulation/config.py

```diff
@@ -103,6 +103,11 @@ class ModelConfig:
             built.
         batch_size: number of columns to batch the ml prediction by. May reduce
             memory use.
+        enforce_precpd_conservative: Use a conservation to determine total precip
```

Comment:
These two flags kind of look similar, could they be a bit more specific to the conservation scheme being used? ex. `..._strict_toa, ..._from_column_water_change` or something to that effect?